From 27f210c863d3349d41e366fb7573e9dcc0708e3d Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Fri, 4 Sep 2020 09:04:29 +0100
Subject: [PATCH] ARM: dts: Limit BT modem baud rate on 3B

The 3B doesn't have the flow control signals connected to the BT modem,
which limits the maximum usable baud rate to below 1 Mbaud. Use
921600 as a relatively safe default value, but the krnbt_baudrate
parameter can still override it.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/bcm2710-rpi-3-b.dts | 4 ++++
 1 file changed, 4 insertions(+)

--- a/arch/arm/boot/dts/bcm2710-rpi-3-b.dts
+++ b/arch/arm/boot/dts/bcm2710-rpi-3-b.dts
@@ -117,6 +117,10 @@
 	status = "okay";
 };
 
+&bt {
+	max-speed = <921600>;
+};
+
 &spi0 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&spi0_pins &spi0_cs_pins>;
